index.html

```diff
@@ -232,7 +232,7 @@ <h2>Core Concept</h2>
              list of boxes at the top of the image with two of them in
              red depicting revoked credentials. Text beside the boxes to the
              right reads 16 kilobytes. An depiction of the boxes being
-             ZLIB compressed into a cylinder on the bottom of the page shows
+             GZIP compressed into a cylinder on the bottom of the page shows
```

Comment:
I have been using https://www.npmjs.com/package/pako

Can we add an informative reference to an implementation you have in mind?

I have seen some really nasty issues with non-deterministic / non-idempotent compression before, and would prefer to avoid accidentally taking a dependency like that.

Reply:
@OR13  DB is using gzip/ungzip from pako for the vc-revocation-list 2020 implementation. 
Here's a gist of the example encodedList value for that
https://gist.github.com/JSAssassin/86f92c6cbdd712e6a6db5f78fd31ba6d. 
Is there anything else you'd like to see?

Reply:
can you link me to the npm package you are using? / unit test for it?

Reply:
@OR13 this is the test suite I am using https://github.com/digitalbazaar/status-list-2021-test-suite/tree/initial, even though the name of the test suite is `status-list-2021`, since 2021 stuff has not been implemented yet it's using endpoints that uses the vc-revocation-list 2020 implementation.